project/sites/parolecomni/themes/parolecomni_theme/src/scss/components.scss

```diff
@@ -17,21 +16,27 @@
   li:last-child {
     border-right: none;
   }
-}
 
-#main-menu .menu-link, #main-menu .menu .active {
-  color: black;
+  .menu-link, #main-menu .menu .active {
+    color: #333;
```

Comment:
It is better to use variables for colours where you can. e.g. $charcoal instead of #333